com.obsproject.Studio.Plugin.BackgroundRemoval.yaml

```
@@ -0,0 +1,67 @@
+id: com.obsproject.Studio.Plugin.BackgroundRemoval
+branch: stable
+runtime: com.obsproject.Studio
+runtime-version: stable
+sdk: org.kde.Sdk//6.4
+build-extension: true
+separate-locales: false
+appstream-compose: false
+build-options:
+  prefix: /app/plugins/BackgroundRemoval
+cleanup:
+  - /include
+  - /share/opencv4
+  - /lib/cmake
+  - /bin
+modules:
+  # onnxruntime has a bash+python+cmake build script - just download/use pre-compiled library
+  # https://onnxruntime.ai/docs/build/inferencing.html
+  - name: onnxruntime
+    buildsystem: simple
+    sources:
+      - type: archive
+        url: https://github.com/microsoft/onnxruntime/releases/download/v1.14.1/onnxruntime-linux-x64-1.14.1.tgz
+        sha256: 010585f534d822bf02d54c7cfc53b57aa3eb0cf34f8fb7503ee519ed18e25d29
+        x-checker-data:
+            type: rotating-url
+            url: https://github.com/microsoft/onnxruntime/releases/latest
+            pattern: "https://github.com/microsoft/onnxruntime/releases/tag/v([0-9.]+)"
+    build-commands:
+      - mkdir -p ${FLATPAK_DEST}/include ${FLATPAK_DEST}/lib
+      - mv include/* ${FLATPAK_DEST}/include
+      - mv lib/* ${FLATPAK_DEST}/lib
+  # https://docs.opencv.org/4.x/db/d05/tutorial_config_reference.html
+  - name: opencv
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+      - "-DBUILD_LIST=core,imgproc"
+    sources:
+      - type: archive
+        url: https://github.com/opencv/opencv/archive/refs/tags/4.7.0.tar.gz
+        sha256: 8df0079cdbe179748a18d44731af62a245a45ebf5085223dc03133954c662973
+        x-checker-data:
+            type: rotating-url
+            url: https://github.com/opencv/opencv/releases/latest
+            pattern: "https://github.com/opencv/opencv/releases/tag/v([0-9.]+)"
+  - name: obs-backgroundremoval
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DUSE_SYSTEM_OPENCV=ON
+      - -DUSE_SYSTEM_ONNXRUNTIME=ON
+      - -DLINUX_PORTABLE=OFF
+    post-install:
+      - install -Dm644 --target-directory=${FLATPAK_DEST}/share/metainfo ${FLATPAK_ID}.metainfo.xml
+      - appstream-compose --basename=${FLATPAK_ID} --prefix=${FLATPAK_DEST} --origin=flatpak
+        ${FLATPAK_ID}
+    sources:
+      - type: archive
+        url: https://github.com/royshil/obs-backgroundremoval/archive/refs/tags/v0.5.15.tar.gz
+        sha256: bae4563374d97512ee7495b8e176204c3961af7f2e0001ce7a0d15f710016091
+        x-checker-data:
+            type: rotating-url
+            url: https://github.com/royshil/obs-backgroundremoval/releases/latest
+            pattern: "https://github.com/royshil/obs-backgroundremoval/releases/tag/v([0-9.]+)"
+      - type: file
+        path: com.obsproject.Studio.Plugin.Backgroundremoval.metainfo.xml
```

```suggestion
        path: com.obsproject.Studio.Plugin.BackgroundRemoval.metainfo.xml
```

Backgroundremoval should be BackgroundRemoval since the file got renamed in a recent commit.